### ISSUE TITLE
Failed to load resource: the server responded with a status of 404 (Not Found)

### ISSUE THREAD

**Issue by kosvrouvas** [rishipr/mern-auth#28]
On Windows **only**, all of a sudden, I get that api/users/login:1  is missing.

Both on a clean install (cloning this repo) and my app.

Did anything change on a dependency maybe?

MacOS works OK.

**Comment by kamranesmaeili** [comment]
Hi @kosvrouvas, if you have managed to send a request from Postman without any issues, this error is then only related to your server.js file. I have had the same problem but after I changed my client proxy (in package.json) to http://localhost:5000 and rebuilt the project it started working. Please also have a look [here](https://stackoverflow.com/questions/55152043/im-getting-404-api-not-found) as this might be useful to you:

**Comment by Sang-Tonsing** [comment]
@kamranesmaeili i have the same error but the answer which you describe above doesn't solve the problem... Is there any other way

**Comment by kamranesmaeili** [comment]
@Sang-Tonsing can't really say where the problem is coming from (although it's most likely from your package.json or server.js) but would recommend looking into this step by step. Maybe use `console.log(userData)` in your `authActions.js` and your api/login to see if the userData are correctly being passed through and see where it stops working.

**Comment by Sang-Tonsing** [comment]
But, since api/users/login is in routes repository aren't we suppose to include the whole path?

**Comment by kamranesmaeili** [comment]
Yes sorry, that's what I meant, to check your API/users/login and see if the code is being executed at all. If not (and your axios/actions work fine), then check your server settings and client package.json as essentially you are sending via axios the right data but your server doesn't find the login API. This can sometimes be due to a wrong proxy port or other settings.

**Comment by Sang-Tonsing** [comment]
Okay, will look into it...

Its not related but can you please help me out here, I wanted to just quit because I have been scratching my head for the past 2 days trying to solve it. I have described the problem here in stack overflow. Thanks alot
https://stackoverflow.com/questions/67253820/fetch-data-from-back-end-to-front-end-after-getting-it-in-router-from-database
